### DISCUSSION TITLE
Custom time input

### DISCUSSION THREAD

**Discussion by lorypelli** [mantinedev/mantine#2640] [Feature requests]
I don't remember if I already asked for it, I tried to search it but I didn't find anything, so I'm remaking the feature request: a custom time input with hours, minutes and seconds will be good

**Comment by rtivital** [comment]
The library already provides TimeInput component – https://mantine.dev/dates/time-input/

**Reply by lorypelli** [reply]
Not a time selector, you need to type it manually
